helm/ingress-nginx/values.yaml

```diff
@@ -736,7 +772,15 @@ controller:
       #   memory: 20Mi
     patchWebhookJob:
       securityContext:
```

Comment:
this one is the containerSecurityContext, right? maybe write a comment above.

Reply:
This one is the container-level security context for the webhook patch job only. See `createSecretJob.securityContext` for the certificate creation job's security context.

Reply:
I see. I'd add the comment though.

Reply:
kk, gonna do so here and in upstream.